commands/validations.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"fmt"
+	"strconv"
+)
+
+// ContextualAtoi cleans the error output of Atoi calls
+func ContextualAtoi(s, resource string) (int, error) {
+	n, err := strconv.Atoi(s)
+	if err == nil {
+		if n < 0 {
+		    return 0, fmt.Errorf("expected %d to be a postive integer", n)
```

Comment:
Typo in `postive`, should be `positive`:
```suggestion
		    return 0, fmt.Errorf("expected %d to be a positive integer", n)
```

Reply:
@ChiefMateStarbuck Not sure if the diff is coming out weird for me or not, but it looks like this typo is still here. Could you fix the typo here as well (assuming it's not already fixed?)